pyaedt/filtersolutions_core/attributes.py

```diff
@@ -0,0 +1,1559 @@
+        self._dll.getStandardDelayEquCut.argtype = POINTER(c_bool)
+        self._dll.getStandardDelayEquCut.restype = c_int
+
+        self._dll.setDelayEquCutoffAttenuationdB.argtype = c_char_p
+        self._dll.setDelayEquCutoffAttenuationdB.restype = c_int
+        self._dll.getDelayEquCutoffAttenuationdB.argtypes = [c_char_p, c_int]
+        self._dll.getDelayEquCutoffAttenuationdB.restype = c_int
+
+    @property
+    def filter_type(self) -> FilterType:
+        """Type (mathematical formulation) of the filter. The default is ``BUTTERWORTH``.
+        The ``FilterType`` enum provides a list of all types.
+
+        Returns
+        -------
+        :enum:`FilterType`
+        """
+        type_string = self._dll_interface.get_string(self._dll.getFilterType)
+        return self._dll_interface.string_to_enum(FilterType, type_string)
+
+    @filter_type.setter
+    def filter_type(self, filter_type: FilterType):
+        if filter_type:
+            string_value = self._dll_interface.enum_to_string(filter_type)
+            self._dll_interface.set_string(self._dll.setFilterType, string_value)
+
+    @property
+    def filter_class(self) -> FilterClass:
+        """Class (band definition) of the filter. The default is ``LOW_PASS``.
+        The ``FilterClass`` enum provides a list of all classes.
+
+        Returns
+        -------
+        :enum:`FilterClass`
+        """
+        type_string = self._dll_interface.get_string(self._dll.getFilterClass)
+        return self._dll_interface.string_to_enum(FilterClass, type_string)
+
+    @filter_class.setter
+    def filter_class(self, filter_class: FilterClass):
+        if filter_class:
+            string_value = self._dll_interface.enum_to_string(filter_class)
+            self._dll_interface.set_string(self._dll.setFilterClass, string_value)
+
+    @property
+    def filter_implementation(self) -> FilterImplementation:
+        """Technology used to implement the filter. The default is ``LUMPED``.
+        The ``FilterImplementation`` enum provides a list of all implementations.
+
+        Returns
+        -------
+        :enum:`FilterImplementation`
+        """
+        type_string = self._dll_interface.get_string(self._dll.getFilterImplementation)
+        return self._dll_interface.string_to_enum(FilterImplementation, type_string)
+
+    @filter_implementation.setter
+    def filter_implementation(self, filter_implementation: FilterImplementation):
+        if filter_implementation:
+            string_value = self._dll_interface.enum_to_string(filter_implementation)
+            self._dll_interface.set_string(self._dll.setFilterImplementation, string_value)
+
+    @property
+    def diplexer_type(self) -> DiplexerType:
+        """Type of diplexer topology. This property is only applicable to lumped filters.
+        The default is ``HI_LO`` for the ``DIPLEXER_1`` filter class.
+        The default is ``BP_BS`` for the ``DIPLEXER_2`` filter class.
```

Comment:
You might want to format these two lines in a bulleted list (with blank lines before the first and after the last) to keep all this text from being placed in one paragrah.